provider/service_resources.go

```diff
@@ -34,7 +34,24 @@ func resourceService() *schema.Resource {
 			"service_tier": {
 				Type:     schema.TypeInt,
 				Optional: true,
-				Default: 5,
+				Default:  5,
+			},
+			"owner": {
+				Type:     schema.TypeList,
```

Comment:
I wanted to turn this into two string fields (`owner_id` and `owner_name`) but all other resources appear to map directly to their api calls without any translation and I didn't want to go against the grain (ex: `services` in `functionality_resource.go`).

Reply:
Yeah, this can probably be more ergonomic but I agree with previous form here.